monkey/monkey_island/cc/server_utils/file_utils.py

```diff
@@ -79,16 +85,15 @@ def _get_file_descriptor_for_new_secure_file_linux(path: str) -> int:
 def _get_file_descriptor_for_new_secure_file_windows(path: str) -> int:
     try:
         file_access = win32file.GENERIC_READ | win32file.GENERIC_WRITE
-        # subsequent open operations on the object will succeed only if read access is requested
-        file_sharing = win32file.FILE_SHARE_READ
+        file_sharing = win32file.FILE_SHARE_READ | win32file.FILE_SHARE_WRITE
```

Comment:
Why this change?

Reply:
From the documentation:

Reply:
From the documentation:
> Enables subsequent open operations on a file or device to request write access.
> Otherwise, other processes cannot open the file or device if they request write access.

It's a bit ambiguous as to whether or not the current process can open for writing without `win32file.FILE_SHARE_WRITE`